src/common/src/catalog/mod.rs

```diff
@@ -125,6 +125,57 @@ impl From<TableId> for u32 {
     }
 }
 
+#[derive(Clone, Copy, Debug, Display, Default, Hash, PartialOrd, PartialEq, Eq, Ord)]
+pub struct TableIdx {
+    pub table_idx: u32,
+}
```

Comment:
It seems `ColumnIdx` is more suitable than `TableIdx`.

Reply:
You are absolutely right. I changed that.